src/main/java/mate/academy/spring/MainApp.java

```diff
@@ -41,6 +52,32 @@ public static void main(String[] args) throws SQLException {
             System.out.println("Email = " + user.getEmail());
             System.out.println();
         }
+        List<Author> foundedAuthors = authorService.findByName("Taras");
+        for (Author author: foundedAuthors) {
+            System.out.println("Id = " + author.getAuthorId());
+            System.out.println("First Name = " + author.getName());
+            System.out.println("Last Name = " + author.getSurname());
+            System.out.println();
+        }
+
+        authorService.findByNameAndSurname("Taras", "Shevchenko");
+        bookService.findByTitle(daisyJonesBook.getTitle());
```

Comment:
```suggestion
        Book daisyJonesBook = bookService.findByTitle("Daisy Jones");
```
@TarasHryts Передавай сюди частину назви... користувач може не пам'ятати назву повністю 
І старайся завжди використати результат виконання методу

Reply:
done